dev/checkstyle/style.xml

```diff
@@ -32,5 +32,14 @@
             <property name="offCommentFormat" value="&lt;editor-fold defaultstate=&quot;collapsed&quot; desc=&quot;Generated Code&quot;&gt;"/>
             <property name="onCommentFormat" value="&lt;/editor-fold&gt;"/>
         </module>
+
+        <module name="ModifierOrder"/>
+        <module name="RedundantModifier"/>
+
+        <module name="SimplifyBooleanExpression"/>
+        <module name="SimplifyBooleanReturn"/>
+
+        <module name="ArrayTypeStyle"/>
+        <module name="UpperEll"/>
```

Comment:
I'm little lazy...what is this one?

Reply:
Incorrect: `1l`. Correct: `1L`. Rationale: `l` is similar to `1` in some fonts.

Reply:
thank you